pkg/apis/capabilities/v1alpha1/zz_generated.openapi.go

```diff
@@ -11,27 +11,27 @@ import (
 
 func GetOpenAPIDefinitions(ref common.ReferenceCallback) map[string]common.OpenAPIDefinition {
 	return map[string]common.OpenAPIDefinition{
-		"github.com/3scale/3scale-operator/pkg/apis/capabilities/v1alpha1.API":               schema_pkg_apis_capabilities_v1alpha1_API(ref),
```

Comment:
How have you generated the openapi autogenerated files?

If I generate them I get changes reverting back to the full path.

In my case I'm using `operator-sdk generate openapi`

```
msoriano@localhost:~/go/src/github.com/3scale/3scale-operator (monitoring-tools)$ operator-sdk version
operator-sdk version: "v0.15.2", commit: "ffaf278993c8fcb00c6f527c9f20091eb8dd3352", go version: "go1.13.3 linux/amd64"
```

Reply:
Same  operator-sdk version

```
$ operator-sdk version
operator-sdk version: "v0.15.2", commit: "ffaf278993c8fcb00c6f527c9f20091eb8dd3352", go version: "go1.13.3 linux/amd64"
```

I tried again and it keeps the changes. They are not reverted back as in your case. My guess, this is because my workspace is not included in GOPATH, (go modules new way of working) and in your case, the workspace is included in GOPATH, hence the module name has to be included. 

TBH, I do not even know what is the use of `operator-sdk generate openapi`. CRD's and openapi validation is generated by `operator-sdk generate crds`. What is the use of `pkg/apis/capabilities/v1alpha1/zz_generated.openapi.go`?

Reply:
For what I've seen, it contains the Go representation of the validation block. Is the Go Kind type's OpenAPI model. It seems it's useful because it allows you to create a full OpenAPI Specification and generate a client from it.

Reply:
We don't use it in any way, right?